src/foam/core/Validation.js

```diff
@@ -141,6 +141,15 @@ foam.CLASS({
             errorString: `Please enter a ${this.label} with at most ${this.maxLength} character${this.maxLength>1?'s':''}`
           });
         }
+        if ( this.required && ! foam.Number.isInstance(this.maxLength || this.minLength) ) {
```

Comment:
```suggestion
        if ( this.required && ! foam.Number.isInstance(this.maxLength) && ! foam.Number.isInstance(this.minLength) ) {
```

Right now you're or-ing two values that may or may not be numbers. Which I guess works due to short-circuiting, but `0 || undefined` evaluates to `undefined`, so it wouldn't work for that case. But then again, I don't see why maxLength would ever be 0.